### DISCUSSION TITLE
Tex syntax not working in app.

### DISCUSSION THREAD

**Discussion by DogancanYr** [community/community#121110] [Mobile]
### Select Topic Area

Bug

### Body

![Screenshot_2024-04-25-17-38-10-976-edit_com github android](https://github.com/community/community/assets/90002969/ebc26ab6-64ea-4344-b1c7-9b840d1b405d)

**Comment by HenSquared** [comment]
Thanks @DogancanYr , I have opened an issue on this. We are aware of this issue and the cause of it. If we require other information, we'll update you here.

**Reply by DogancanYr** [reply]
When will it be fixed?

**Comment by DogancanYr** [comment]
When will it be fixed?

**Comment by DogancanYr** [comment]
Is there a bug priority list?
